sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveInspectorSuite.scala

```diff
@@ -93,7 +92,6 @@ class HiveInspectorSuite extends FunSuite with HiveInspectors {
   val row = data.map(_.eval(null))
   val dataTypes = data.map(_.dataType)
 
-  import scala.collection.JavaConversions._
```

Comment:
Most of the code in Spark SQL uses JavaConversions... I don't think performance is an issue here or anything.